### PR TITLE
lts: 2.7.5: Release notes for mbedTLS

### PR DESCRIPTION
**Pull request by ceolin**
Informational notes about mbedTLS module update.

### CODE REVIEW

**Review by cfriedt — COMMENTED** [review]
Looks reasonable enough to me. I wanted to ensure that @microbuilder had an opportunity to look at this as well. Kevin, do you have any suggestions or change requests?

File: doc/releases/release-notes-2.7.rst [hunk 1]
```diff
@@ -19,6 +19,23 @@ release:
    * :github:`issuenumber` - issue title
 
 
+mbedTLS
+*******
+
+Moving mbedTLs to 2.28.x series (2.28.3 precisely). This is a LTS release
```
Comment (microbuilder) [inline doc/releases/release-notes-2.7.rst:25]: mbedTLS

**Review by microbuilder — COMMENTED** [review]
Looks fine to me, other than a minor nit with MbedTLS. I think maybe it should be an upper case M as well, but less sure about that.

File: doc/releases/release-notes-2.7.rst (hunk 1 above)
Comment (ceolin) [reply doc/releases/release-notes-2.7.rst]: Thanks !

File: doc/releases/release-notes-2.7.rst [hunk 2]
```diff
@@ -19,6 +19,23 @@ release:
...
+
+Moving mbedTLS to 2.28.x series (2.28.3 precisely). This is a LTS release
+that will be supported with bug fixes and security fixes until the end of 2024.
+
+Detailed information can be found in:
+https://github.com/Mbed-TLS/mbedtls/releases/tag/v2.28.3
+https://github.com/zephyrproject-rtos/zephyr/issues/56071
+
+This version is incompatible with TF-M and because of this TF-M is no longer
+support in Zephyr LTS. If TF-M is required it can be manually added back
```
Comment (mrfuchs) [inline doc/releases/release-notes-2.7.rst:33]: 'supported' (instead of 'support')?

Comment (ceolin) [reply doc/releases/release-notes-2.7.rst]: done :)

**Review by microbuilder — APPROVED** [review]

**Review by mrfuchs — APPROVED** [review]

**Review by cfriedt — APPROVED** [review]
@ceolin - will there be a 2nd PR for CVEs?

**Comment by cfriedt** [comment]
@stephanosio - can you please use your merge superpowers? I could create a ticket, but meh.

**Review by stephanosio — APPROVED** [review]

**Pull request merged**